src/node/internal/internal_http_incoming.ts

```diff
@@ -24,12 +40,38 @@ export class IncomingMessage extends Readable implements _IncomingMessage {
   httpVersionMajor = 1;
   httpVersionMinor = 1;
   httpVersion: string = '1.1';
+  complete = false;
+  rawHeaders: string[] = [];
+  rawTrailers: string[] = [];
```

Comment:
Prefer just to remove anything related to trailers in the impl here (likewise the `kTrailersDistinct`, `kTrailersCount`, `kTrailers`, etc.